sql/core/src/test/scala/org/apache/spark/sql/SparkSessionBuilderSuite.scala

```diff
@@ -36,18 +38,22 @@ class SparkSessionBuilderSuite extends SparkFunSuite {
     initialSession.sparkContext
   }
 
+  override def afterEach(): Unit = {
+    // This suite should not interfere with the other test suites.
+    SparkSession.clearDefaultSession()
+    SparkSession.clearActiveSession()
```

Comment:
We need to update the unit test case because SparkContext requires `spark.master`. I'll update like that.
```
    if (!_conf.contains("spark.master")) {
      throw new SparkException("A master URL must be set in your configuration")
    }
```